packages/block-editor/src/layouts/flow.js

```diff
@@ -72,6 +74,7 @@ export default {
 								onChange( {
 									...layout,
 									wideSize: nextWidth,
+									useGlobalPadding: true,
```

Comment:
Now that we no longer have a toggle for using `useGlobalPadding` do we need the boolean to be stored in the `layout` object? I was wondering if we can instead infer whether to use global padding at render time in `layout.php`.

Reply:
Good point, I don't think we need this anymore!